content/docs/command-reference/exp/apply.md

```diff
@@ -6,7 +6,7 @@ Put the results from an [experiment](/doc/command-reference/exp) in the
 ## Synopsis
 
 ```usage
-usage: dvc exp apply [-h] [-q | -v] experiment
+usage: dvc exp apply [-h] [-q | -v] [--no-force] experiment
```

Comment:
I don't think `--no-force` needs anything beyond what's listed in the help text

```
optional arguments:
...
  --no-force     Fail if this command would overwrite conflicting changes.
```

Reply:
Yup, OK I'll add that. Though 

> overwriting any conflicting workspace changes with --force should be the default behavior
From https://github.com/iterative/dvc/pull/5360

touches on some recent discussion about `merge --theirs` and how it can end up mixing up with other workspace changes (https://github.com/iterative/dvc.org/issues/2397#issuecomment-822890375). We could clarify on that in this PR perhaps.

Reply:
Updated in df1430f1.